subcommands/backup/backup.go

```diff
@@ -38,20 +38,24 @@ func init() {
 	subcommands.Register(func() subcommands.Subcommand { return &Backup{} }, subcommands.AgentSupport, "backup")
 }
 
-type excludeFlags []string
+type flagList []string
 
-func (e *excludeFlags) String() string {
+// Called by the flag package to print the default / help.
+func (e *flagList) String() string {
 	return strings.Join(*e, ",")
 }
```

Comment:
i thought we were going to go with the `-tag one,two,three` instead of `-tag one -tag two -tag three`, but anyway it's OK for me.  My only "gripe" is that if we go with multiple -tags then this is wrong, but on the other hand i don't see how to fix it.

just wanted to point it out, not a big deal.